ftp_blueprints/download_file.py

```diff
@@ -55,6 +55,13 @@ def extract_file_name_from_source_full_path(source_full_path):
     return os.path.basename(source_full_path)
 
 
+def is_dot_directory(full_path):
+    """
+    Determine if the provided folder_name is a dot directory.
+    """
+    return all(char == '.' for char in full_path)
```

Comment:
Do we want to exclude all dot directories or just the relative ones (`../`)?

Reply:
This specific bug is caused because `.` and `..` continually get added to the folders list.

In theory we would like to omit the relative path directories as well but to minimize the risk of introducing new bugs i opted for the addressing this specific error for now